src/triggers.ts

```diff
@@ -344,15 +351,19 @@ export function createTriggersSystem(
 
   // ON_INPUT_ACTION
   function initOnInputActionTrigger(entity: Entity) {
-    const pointerEvent = PointerEvents.getOrNull(entity)
+    const pointerEvent = PointerEvents.getMutableOrNull(entity)
 
     const opts = {
-      button:
-        pointerEvent?.pointerEvents[0].eventInfo?.button ||
-        InputAction.IA_PRIMARY,
-      ...(pointerEvent === null ? { hoverText: 'Press' } : {}),
+      button: InputAction.IA_PRIMARY,
```

Comment:
should we keep the `pointerEvent?.pointerEvents[0].eventInfo?.button` for when the users define a button from the inspector?

Reply:
Is that possible ? cc @nearnshaw

Reply:
sorry I didn't fully understand the question
what is a `button` in this context? I assume it's an internal definition of asset-packs

Reply:
The tricky thing I think is that the PointerEvents defines the hover feedback, but not the actual interaction of the event, right?

I don't know exactly how the asset-packs library deals with that, maybe there is something behind the scenes that ensures there's consistency

Reply:
When you define a [InputAction trigger](https://github.com/decentraland/asset-packs/blob/main/src/triggers.ts#L346-L366), the `initOnInputActionTrigger` check if the entity has a defined `PointerEvent` to keep that value, otherwise it will define a new `PointerEvent` with a default value `(Press E)`.

Reply:
> When you define a [InputAction trigger](https://github.com/decentraland/asset-packs/blob/main/src/triggers.ts#L346-L366), the initOnInputActionTrigger check if the entity has a defined PointerEvent to keep that value, otherwise it will define a new PointerEvent with a default value (Press E).

Right now its creating a new one with an empty hoverText, that ends with an `E Interact` because the explorer fallbacks to that text.